scripts/lib/links/FileBatch.ts

```diff
@@ -14,7 +14,7 @@ import { globby } from "globby";
 
 import { Link, File } from "./LinkChecker";
 import FILES_TO_IGNORES from "./ignores";
-import { getMarkdownAndAnchors, addLinksToMap } from "./markdown";
+import { getMarkdownAndAnchors, addLinksToMap } from "./extractLinks";
```

Comment:
This link checker stuff was tricky due to bad abstractions before. See https://github.com/Qiskit/documentation/pull/628. I recommend merging that PR first even though it will result in a merge conflict. It should remove all changes to this file because it makes the code in this file generic.

You would only need to in this PR change `extractLinks.ts`. In the long term, you'd want it to:

- set `anchors` to `[]` because we don't expect anything to link to `objects.inv`, so it doesn't make sense to populate
- set `links` to all the URLs contained in the file that we need to validate. But for now, you could simply return `[]`.

Reply:
Thanks, resolving as #628 is merged now